Source/Cypher/app/src/main/java/com/semaphore_soft/apps/cypher/ConnectionLobbyActivity.java

```diff
@@ -99,4 +166,37 @@ private void addTestPlayers()
             playersList.add(playerID);
         }
     }
+
+    private class ResponseReceiver extends BroadcastReceiver
```

Comment:
I expect we can, if not will need to, reuse this broadcast receiver in another activity. Suggest we define it in its own class in a networking package rather than leaving it as a private inner class of the connection lobby activity.